common/religion/religions/fel_group.txt

```diff
@@ -239,6 +253,11 @@
 			doctrine = tenet_pursuit_of_power
 			doctrine = tenet_elemental_syncretism
 			doctrine = tenet_warmonger
+
+			#Magic
+			doctrine = doctrine_elemental_water_magic_accepted
```

Comment:
The Light magic may be a crime after the draenei.

Reply:
Done